MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift

```diff
@@ -14,11 +14,16 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
 
     // MARK: - Variables
     private lazy var nowPlayingViewModel = NowPlayingViewModel(nowPlayingRepository: NowPlayingRepository(), delegate: self)
-
+    private lazy var upcomingMoviesViewModel = UpcomingMoviesViewModel(upcomingMoviesRepository: UpcomingMoviesRepository(), delegate: self)
```

Comment:
so in the future, you can just have 1 viewModel that will handle the getch movie functions

Reply:
understood